webknossos/webknossos/client/api_client/errors.py

```diff
@@ -13,9 +13,7 @@ def message_for_response_body(self, response: httpx.Response) -> str:
         )
         return f"Got response status {response.status_code} with body{shortened_label}: {response_str[0:response_limit_chars]}"
 
-    check_version_hint = """If this is unexpected, please double-check your WEBKNOSSOS URL and credentials.
```

Comment:
Do we still have a warning about the version somewhere?

Reply:
I talked with @fm3 about this error message. Version mismatch errors are reported within the error messages of the backend now. This version mismatch error was removed as it caused some confusion for users.